.github/workflows/ee.yml

```diff
@@ -0,0 +1,109 @@
+---
+name: execution environment
+on:
+  # Run CI against all pushes (direct commits, also merged PRs), Pull Requests
+  push:
+    branches:
+      - main
+      - stable-*
+  pull_request:
+  # Run CI once per day (at 04:45 UTC)
+  # This ensures that even if there haven't been commits that we are still testing against latest version of ansible-builder
+  schedule:
+    - cron: '45 4 * * *'
+
+env:
+  NAMESPACE: community
+  COLLECTION_NAME: crypto
+
+jobs:
+  build:
+    name: Build and test EE (Ⓐ${{ matrix.runner_tag }})
+    strategy:
+      matrix:
+        runner_tag:
```

Comment:
do they not have `stable-2.13` yet?

Reply:
There isn't (https://quay.io/repository/ansible/ansible-runner?tab=tags&tag=latest). I guess it will show up once 2.13.0 has been released.